snforge_std/src/cheatcodes.cairo

```diff
@@ -110,6 +110,17 @@ fn stop_elect(target: CheatTarget) {
     handle_cheatcode(cheatcode::<'stop_elect'>(inputs.span()));
 }
 
+
+/// Mocks contract call to a `function_selector` of a contract at the given address, for `n_times` first calls that are made
+/// to the contract.
+/// A call to function `function_selector` will return data provided in `ret_data` argument.
+/// An address with no contract can be mocked as well.
+/// An entrypoint that is not present on the deployed contract is also possible to mock.
+/// Note that the function is not meant for mocking internal calls - it works only for contract entry points.
+/// - `contract_address` - target contract address
+/// - `function_selector` - selector of the function in a contract at the `contract_address` that will be mocked
```

Comment:
Should it include anything about possible obtaining it from the `selector!` macro?

Reply:
yeah i think i'll include that